libraries/Microsoft.Bot.Builder.AI.LUIS/LuisRecognizerOptionsV3.cs

```diff
@@ -0,0 +1,158 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder.TraceExtensions;
+using Microsoft.Bot.Schema;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Bot.Builder.AI.Luis
+{
+    public class LuisRecognizerOptionsV3 : LuisRecognizerOptions
+    {
+        /// <summary>
+        /// The value type for a LUIS trace activity.
+        /// </summary>
+        public const string LuisTraceType = "https://www.luis.ai/schemas/trace";
+
+        /// <summary>
+        /// The context label for a LUIS trace activity.
+        /// </summary>
+        public const string LuisTraceLabel = "LuisV3 Trace";
+
+        public LuisRecognizerOptionsV3(LuisApplication application)
+        : base(application)
+        {
+        }
+
+        public LuisV3.LuisPredictionOptions PredictionOptions { get; set; } = new LuisV3.LuisPredictionOptions();
+
+        internal override async Task<RecognizerResult> RecognizeInternalAsync(ITurnContext turnContext, HttpClient httpClient, CancellationToken cancellationToken)
+        {
+            BotAssert.ContextNotNull(turnContext);
+            if (turnContext.Activity.Type != ActivityTypes.Message)
+            {
+                return null;
+            }
+
+            var options = PredictionOptions;
+            var utterance = turnContext.Activity?.AsMessageActivity()?.Text;
+            RecognizerResult recognizerResult;
+            JObject luisResponse = null;
+
+            if (string.IsNullOrWhiteSpace(utterance))
+            {
+                recognizerResult = new RecognizerResult
+                {
+                    Text = utterance,
+                    Intents = new Dictionary<string, IntentScore>() { { string.Empty, new IntentScore() { Score = 1.0 } } },
```

Comment:
Hi, it seems V3 is a new one. So will it be changed to align with luis without breaking existing dependencies?

For example, when one queries ' ' in luis, it returns `{"query":null,"intents":[],"entities":[]}` instead of an Intent with string.Empty Intent.

Related https://github.com/microsoft/botbuilder-dotnet/issues/2174

Thanks~